sample_apps/simple_app_python/example_app.py

```diff
@@ -123,6 +145,7 @@ def do_main(args) -> bool:
 
     public_key_alg = "rsa-2048"
     symmetric_key_alg = "aes-256-cbc-hmac-sha256"
+    whoami = 'unknown'
```

Comment:
this is not needed, just save the cert+key in the application dir

Reply:
Simplified this logic. Now, cert / key-files are written out when 'server' app runs the `get-certified` operation.